### PR TITLE
added test.py

### PR DESCRIPTION
**Pull request by mysticetii**
Added test.py for unit testing functionalities.
Added devMode parameter in graphqlwfs function

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -4,10 +4,11 @@
...
 
 def fetchFeaturesFromWFS(count, typeNames, filters):
-    OS_KEY = os.getenv('OS_KEY', '????????')
+    #OS_KEY = os.getenv('OS_KEY', '????????')
     #Edit WFS API Endpoint address here
-    wfsApiBaseUrl = "https://api.os.uk/features/v1/wfs?service=wfs&request=GetFeature&key={}&version=2.0.0&outputformat=geoJSON".format(
-        OS_KEY)
+    #wfsApiBaseUrl = "https://api.os.uk/features/v1/wfs?service=wfs&request=GetFeature&key={}&version=2.0.0&outputformat=geoJSON".format(
+    #    OS_KEY)
+    wfsApiBaseUrl = "https://api.os.uk/features/v1/wfs?service=wfs&request=GetFeature&key=59vP4agMAycoI32XMjA6wzhBMx3QybCf&version=2.0.0&outputformat=geoJSON"
```
Comment (sebastianovide) [inline main.py:11]: remove hard coding

File: main.py [hunk 2]
```diff
@@ -110,11 +111,16 @@ def resolve_zoomstackNames(self, info, first, Name1):
     <http://flask.pocoo.org/docs/1.0/api/#flask.Flask.make_response>.
 """
 
-
-def graphqlwfs(request):
+def graphqlwfs(request, devMode=False):
```
Comment (sebastianovide) [inline main.py:114]: is devMode part of GCP cloud function ?

**Review by sebastianovide — CHANGES_REQUESTED** [review]
see my comments. Need also to add a note in the readme about how to run the test

File: main.py (hunk 2 above)
Comment (mysticetii) [reply main.py]: devMode has been removed. It was an artifact of a previous attempt to add test.py.

File: main.py (hunk 1 above)
Comment (mysticetii) [reply main.py]: I have removed the hardcoded os_key and added the default key in place of question mark if OS_KEY cannot be retrieved from the environment

File: main.py [hunk 3]
```diff
@@ -4,10 +4,12 @@
 
 
 def fetchFeaturesFromWFS(count, typeNames, filters):
-    OS_KEY = os.getenv('OS_KEY', '????????')
+    # defaults to given key value if OS_KEY cannot be accepted from environment variable.
+    OS_KEY = os.getenv('OS_KEY', '59vP4agMAycoI32XMjA6wzhBMx3QybCf')
```
Comment (sebastianovide) [inline main.py:8]: remove hard coding. (revert it)

**Review by sebastianovide — CHANGES_REQUESTED** [review]
revert main.py

File: main.py (hunk 3 above)
Comment (mysticetii) [reply main.py]: hard coded key has been removed

**Pull request closed without merging**